src/main/scala/firrtl/Emitter.scala

```diff
@@ -92,6 +92,14 @@ class VerilogEmitter extends Emitter {
     }
     case _ => error("Shouldn't be here")
```

Comment:
Just allowing a MatchError would be better here since at least then it will say what you got but shouldn't have....